examples/cms-starter/src/data.ts

```diff
@@ -0,0 +1,293 @@
+import { CollectionItemData, ManagedCollectionField, framer } from "framer-plugin"
+import { assert, generateHash, slugify } from "./utils"
+
+// Plugin keys
+export const PLUGIN_PREFIX = "cms_starter"
+export const LOCAL_STORAGE_LAST_LAUNCH_KEY = `${PLUGIN_PREFIX}.lastLaunched`
+
+export const PLUGIN_COLLECTION_SYNC_REFERENCE_KEY = `collectionSyncReference`
+export const PLUGIN_COLLECTION_SYNC_SLUG_KEY = `collectionSyncSlug`
+
+export type DataSourceFieldType =
+    | "string"
+    | "date"
+    | "image"
+    | "reference"
+    | "richText"
+    | "number"
+    | "boolean"
+    | "enum"
+    | "color"
```

Comment:
Can we not get this from the types exported on framer-plugin? Don't want to need to maintain this, really.

Reply:
We have `CollectionField["type"]`

Reply:
And `ManagedCollectionField`